### ISSUE TITLE
MAE supported. 

### ISSUE THREAD

**Issue by RuoyuFeng** [open-mmlab/mmselfsup#171]
Would you support MAE pretraining in the future? Will it include the transferring to down-stream tasks (object detection, segmentation) ?

Thanks very much.

**Comment by YuanLiuuuuuu** [comment]
Currently, we have already released our unofficial version of MAE pretraining and finetuning, please refer to [MAE](https://github.com/open-mmlab/mmselfsup/tree/feature/mae_experimental/configs/selfsup/mae). As for other downstream tasks, e.g. objection detection and segmentation, we will support it, but will release them in mmseg and mmdet, please stay tuned !

**Comment by RuoyuFeng** [comment]
> Currently, we have already released our unofficial version of MAE pretraining and finetuning, please refer to [MAE](https://github.com/open-mmlab/mmselfsup/tree/feature/mae_experimental/configs/selfsup/mae). As for other downstream tasks, e.g. objection detection and segmentation, we will support it, but will release them in mmseg and mmdet, please stay tuned !

Got it, thanks very much.

**Comment by YuanLiuuuuuu** [comment]
<img width="338" alt="屏幕截图 2022-01-24 143820" src="https://user-images.githubusercontent.com/30762564/150733959-2959852a-c7bd-4d3f-911f-3e8d8839fe67.png">

**Comment by lxtGH** [comment]
Hi! Great work on MAE. @YuanLiuuuuuu However, why the results are litter lower than original MAE repo. Is this because less pretraining epochs ??

**Comment by YuanLiuuuuuu** [comment]
> Hi! Great work on MAE. @YuanLiuuuuuu However, why the results are litter lower than original MAE repo. Is this because less pretraining epochs ??

Yes. Because of the limited computation resource, we only pre-train MAE for 400 epochs. In addition, we also pre-train MAE using the code from the [official repo](https://github.com/facebookresearch/mae) for 400 epochs, and the result is the same as ours.

**Comment by lxtGH** [comment]
Thank you !

**Comment by austinmw** [comment]
Hi, can you also consider: [Masked Autoencoders As Spatiotemporal Learners](https://arxiv.org/abs/2205.09113)

**Comment by fangyixiao18** [comment]
> Hi, can you also consider: [Masked Autoencoders As Spatiotemporal Learners](https://arxiv.org/abs/2205.09113)

Thanks for your reporting, we will consider and discuss it.
